src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationUnitTest.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.data.documentdb.repository;
+
+import com.microsoft.azure.documentdb.IndexingPolicy;
+import com.microsoft.azure.spring.data.documentdb.Constants;
+import com.microsoft.azure.spring.data.documentdb.core.mapping.Document;
+import com.microsoft.azure.spring.data.documentdb.core.mapping.DocumentIndexingPolicy;
+import com.microsoft.azure.spring.data.documentdb.domain.Person;
+import com.microsoft.azure.spring.data.documentdb.domain.Role;
+import com.microsoft.azure.spring.data.documentdb.repository.support.DocumentDbEntityInformation;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.util.Assert;
+
+
+public class DocumentDBAnnotationUnitTest {
+    private DocumentDbEntityInformation<Person, String> personInfo;
+    private DocumentDbEntityInformation<Role, String> roleInfo;
+
+    @Before
+    public void setUp() {
+        personInfo = new DocumentDbEntityInformation<>(Person.class);
+        roleInfo = new DocumentDbEntityInformation<>(Role.class);
+    }
+
+    @Test
+    public void testNoDocumentDBAnnotation() {
```

Comment:
What's the difference between this test and this one: https://github.com/Microsoft/spring-data-documentdb/pull/49/files#diff-cd92904246e1bc32d880298904dbb455R154 ? With the same test purpose?

Reply:
One for UT, and the other for IT, as create document db instance in azure may cost some time @yungez suggest to move the test here from reducing IT time cost.